cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -18,6 +18,20 @@ Existing tools such as `oc` are more operations-focused and require a deep under
 * Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the {product-title} cluster for deployment.
 * Official full support of Node.js and Java components.
-* Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
+* Partial compatibility with languages and frameworks such as Ruby, Perl, PHP, and Python. 
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the {product-title} cluster.
\ No newline at end of file
+* Lists all the available components and services from the {product-title} cluster.
+
+=== Supported languages and corresponding container images
```

Comment:
Each heading needs an explicit anchor ID.

Reply:
fixed, thank you, Kathryn!